apps/agentic-chat/src/components/chat-message-list.tsx

```diff
@@ -5,31 +5,111 @@ import { ScrollArea } from './ui/scroll-area';
 import { cn } from '../lib/utils';
 import { MessageList } from '../types/message';
 import Markdown from 'react-markdown';
+import {
+  ChatMessage,
+  OpenAIToolCall,
```

Comment:
Also this type is deprecated but not super concerned, literally just a type. There is `ToolCall` which is pretty much the same but not exported.
There is also `AiMessage['tool_calls']` but this is a `ToolCall[]` (array), so doesn't really work